src/Pomelo.EntityFrameworkCore.MySql/Utilities/Json/JsonObject`1.cs

```diff
@@ -0,0 +1,212 @@
+﻿using static Newtonsoft.Json.JsonConvert;
+
+// ReSharper disable once CheckNamespace
+namespace System
+{
+  public class JsonObject<T> : IEquatable<JsonObject<T>>, IEquatable<JsonObject>, IEquatable<string>
```

Comment:
I agree, but it's to maintain BC with Pomelo.Data.MySQL (which this package presently imports)